at-paninij-core/at-paninij-proc/src/main/java/org/paninij/proc/factory/AbstractMessageFactory.java

```diff
@@ -76,7 +76,6 @@ public String buildQualifiedClassName() {
         packs.add("javax.annotation.Generated");
         packs.add("org.paninij.runtime.Panini$Future");
         packs.add("org.paninij.runtime.Panini$Message");
-        packs.add(ret.packed());
```

Comment:
Looks like futures and ducks just use `.toString()` on their `TypeMirror`. So if I have an array with generics for instance, the resulting future uses `java.util.List<? extends java.lang.Runnable>[]`, so no import seems to be needed.